app/views/layouts/govuk_publishing_components/preview.html.erb

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
Could the original layout have some logic to deal with the differences? I'm keen to avoid the duplication of layouts.